src/System.Data.SqlClient/src/Configurations.props

```diff
@@ -8,10 +8,10 @@
       netstandard1.3;
       netstandard;
       netfx-Windows_NT;
+      uap-Windows_NT;
```

Comment:
If you want this configuration to apply to `uap10.0.16299` you need to target that.  Right now this package will apply it's `netstandard-Windows_NT` configuration to `uap10.0.16299` and the `uap-Windows_NT` configuration to `uap10.0.16300` (since `uap` has been moved forward to represent that version).

Instead you'll want:
```xml
     <PackageConfigurations>
       netstandard-Unix;
       netstandard-Windows_NT;
       netstandard1.2;
       netstandard1.3;
       netstandard;
       netfx-Windows_NT;
       uap10.0.16299-Windows_NT;
     </PackageConfigurations>
     <BuildConfigurations>
       $(PackageConfigurations)
       uap-Windows_NT;
     </BuildConfigurations>
```
And update the conditions in the src project to treat them the same.